docs/accessibilityinfo.md

```diff
@@ -143,6 +146,83 @@ const styles = StyleSheet.create({
 export default AccessibilityStatusExample;
 ```
 
+</TabItem>
+<TabItem value="typescript">
```

Comment:
General note: even though the TS tab comes after in the document, it is still shown first on the actual webpage (controlled by the TabConstants ordering).